.gitignore

```diff
@@ -1,126 +1,15 @@
-# Created by https://www.gitignore.io/api/flask
-# Edit at https://www.gitignore.io/?templates=flask
-
-### Flask ###
-instance/*
-!instance/.gitignore
-.webassets-cache
-
-### Flask.Python Stack ###
-# Byte-compiled / optimized / DLL files
+#python cache 
 __pycache__/
-*.py[cod]
-*$py.class
-
-# C extensions
-*.so
-
-# Distribution / packaging
-.Python
-build/
-develop-eggs/
-dist/
-downloads/
-eggs/
-.eggs/
-lib/
-lib64/
-parts/
-sdist/
-var/
-wheels/
-pip-wheel-metadata/
-share/python-wheels/
-*.egg-info/
-.installed.cfg
-*.egg
-MANIFEST
-
-# PyInstaller
-#  Usually these files are written by a python script from a template
-#  before PyInstaller builds the exe, so as to inject date/other infos into it.
-*.manifest
-*.spec
-
-# Installer logs
-pip-log.txt
-pip-delete-this-directory.txt
-
-# Unit test / coverage reports
-htmlcov/
-.tox/
-.nox/
-.coverage
-.coverage.*
-.cache
-nosetests.xml
-coverage.xml
-*.cover
-.hypothesis/
 .pytest_cache/
 
-# Translations
-*.mo
-*.pot
-
-# Scrapy stuff:
-.scrapy
-
-# Sphinx documentation
-docs/_build/
-
-# PyBuilder
-target/
-
-# pyenv
-.python-version
-
-# celery beat schedule file
-celerybeat-schedule
-
-# SageMath parsed files
-*.sage.py
-
-# Spyder project settings
-.spyderproject
-.spyproject
-
-# Rope project settings
-.ropeproject
-
-# Mr Developer
-.mr.developer.cfg
-.project
-.pydevproject
-
-# mkdocs documentation
-/site
-
-# mypy
-.mypy_cache/
-.dmypy.json
-dmypy.json
-
-# Pyre type checker
-.pyre/
-
-# End of https://www.gitignore.io/api/flask
-
-### macOS ###
-.DS_Store
-
-### Virtual environment ###
-.venv/
+#VSCode folder
+.vscode
 
-### Configuration files ###
+#configuration files
 .env
 
-# VSCode folder
-.vscode
-
 #vagrant
 .vagrant
 
 #geckodriver binary
-geckodriver.exe
```

Comment:
You should have added chromedriver.exe in its place.

But adding to the .gitignore doesn't affect files that are already being tracked so you now also need to run `git rm --cached chromedriver.exe` to remove it from the repo.